tasks/main.yml

```diff
@@ -3,35 +3,34 @@
 
 - include_vars: "{{ ansible_os_family }}.yml"
 
-- include: "install-{{ ansible_os_family }}.yml"
+- name: Fail when hostname_fqdn is not defined
+  fail:
+    msg: hostname_fqdn must not be empty
```

Comment:
* s/not be empty/must be defined/ ?
* though i do not know which of `assert` or `fail` should be used,  using `assert` with the combined conditionals which check "defined" and "not empty" looks straightforward.